types/vimeo__player/index.d.ts

```diff
@@ -43,13 +44,17 @@ export class Player {
     addCuePoint(time: number, data: VimeoCuePointData): VimeoPromise<string, UnsupportedError | RangeError | Error>;
     removeCuePoint(id: string): VimeoPromise<string, UnsupportedError | InvalidCuePoint | Error>;
     getCuePoints(): VimeoPromise<VimeoCuePoint[], UnsupportedError | Error>;
+    getBuffered(): VimeoPromise<number, Error>;
```

Comment:
[`getBuffered` returns an array of buffered time ranges](https://github.com/vimeo/player.js/#getbuffered-promisearray-error), should probably be `[number, number][]` because it returns an array of tuples.